initrd/bin/config-gui.sh

```diff
@@ -136,6 +141,97 @@ while true; do
         exit 0
       fi
     ;;
+    "R" )
+      CURRENT_OPTION=`grep 'CONFIG_ROOT_DEV=' /tmp/config | tail -n1 | cut -f2 -d '=' | tr -d '"'`
```

Comment:
I think this is where the toggling under Configure Settings menu toggle script fails for non librems since we are `set -e -o pipefail` and the grep fails here?

Reply:
Yeah you're probably right, good catch! :bow: